src/vis/scatter/LabelingOptions.tsx

```diff
@@ -0,0 +1,26 @@
+import { Input, SegmentedControl } from '@mantine/core';
+import * as React from 'react';
+import { ELabelingOptions } from './interfaces';
+
+interface LabelingOptionsProps {
+  callback: (s: ELabelingOptions) => void;
+  currentSelected: ELabelingOptions | null;
+}
+
+export function LabelingOptions({ callback, currentSelected }: LabelingOptionsProps) {
+  return (
+    <Input.Wrapper label="Labels">
+      <SegmentedControl
+        fullWidth
+        size="xs"
+        value={currentSelected}
+        onChange={callback}
+        data={[
+          { label: ELabelingOptions.SELECTED, value: ELabelingOptions.SELECTED },
+          { label: ELabelingOptions.ON, value: ELabelingOptions.ON },
+          { label: ELabelingOptions.OFF, value: ELabelingOptions.OFF },
```

Comment:
I suggest to adapt the label and enums to:

```suggestion
          { label: ELabelingOptions.NEVER, value: ELabelingOptions.NEVER },
          { label: ELabelingOptions.ALWAYS, value: ELabelingOptions.ALWAYS },
          { label: ELabelingOptions.SELECTED, value: ELabelingOptions.SELECTED },
```